runtime/src/main/kotlin/kotlin/native/Text.kt

```diff
@@ -21,12 +21,20 @@ public fun ByteArray.stringFromUtf8() : String {
 @Deprecated("Use decodeToString instead", ReplaceWith("decodeToString(start, start + size)"))
```

Comment:
We'd also like to remove the deprecation replacement for now, until we design the replacement function.

Reply:
+1